errtracker/errtracker.go

```diff
@@ -119,6 +135,7 @@ func Report(snap, errMsg, dupSig string, extra map[string]string) (string, error
 		"CoreSnapdBuildID":   coreBuildID,
 		"Date":               timeNow().Format(time.ANSIC),
 		"Snap":               snap,
+		"PartialDpkgUpdate":  detectPartialDpkgUpdate(),
```

Comment:
I think we can do two things:
- show the flag that .dpkg-new is present
- carry md5 of the actual profile

WDYT?